<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QMathParser</class>
 <widget class="QMainWindow" name="QMathParser">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>781</width>
    <height>535</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QMathParser</string>
  </property>
  <property name="windowIcon">
   <iconset resource="qmathparser.qrc">
    <normaloff>:/qmp/resource/qmp.png</normaloff>:/qmp/resource/qmp.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0" colspan="2">
     <widget class="QLCDNumber" name="lcdres">
      <property name="styleSheet">
       <string notr="true">color: rgb(85, 255, 0);
background-color: rgb(0, 0, 0);</string>
      </property>
      <property name="numDigits">
       <number>16</number>
      </property>
      <property name="segmentStyle">
       <enum>QLCDNumber::Flat</enum>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLineEdit" name="textEdit">
      <property name="font">
       <font>
        <family>STIXGeneral</family>
        <pointsize>16</pointsize>
        <italic>true</italic>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QDockWidget" name="dockWidget_2">
   <property name="windowTitle">
    <string>Input Bar</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>8</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_2">
    <layout class="QGridLayout" name="gridLayout_4">
     <item row="0" column="0">
      <widget class="QGroupBox" name="groupBox_2">
       <property name="title">
        <string>Numbers</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_5">
        <item row="2" column="0">
         <widget class="QPushButton" name="pushButton_35">
          <property name="text">
           <string>1</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QPushButton" name="pushButton_36">
          <property name="text">
           <string>2</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QPushButton" name="pushButton_37">
          <property name="text">
           <string>4</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QPushButton" name="pushButton_38">
          <property name="text">
           <string>5</string>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QPushButton" name="pushButton_39">
          <property name="text">
           <string>6</string>
          </property>
         </widget>
        </item>
        <item row="2" column="2">
         <widget class="QPushButton" name="pushButton_40">
          <property name="text">
           <string>3</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QPushButton" name="pushButton_41">
          <property name="text">
           <string>7</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QPushButton" name="pushButton_42">
          <property name="text">
           <string>8</string>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QPushButton" name="pushButton_43">
          <property name="text">
           <string>9</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0" colspan="3">
         <widget class="QPushButton" name="pushButton_44">
          <property name="text">
           <string>0</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item row="1" column="0" colspan="2">
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>Functions</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0">
         <widget class="QPushButton" name="pushButton_27">
          <property name="text">
           <string>sqrt</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QPushButton" name="pushButton_24">
          <property name="text">
           <string>log2</string>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QPushButton" name="pushButton_12">
          <property name="text">
           <string>sin</string>
          </property>
         </widget>
        </item>
        <item row="0" column="3">
         <widget class="QPushButton" name="pushButton_15">
          <property name="text">
           <string>asin</string>
          </property>
         </widget>
        </item>
        <item row="0" column="4">
         <widget class="QPushButton" name="pushButton_18">
          <property name="text">
           <string>sinh</string>
          </property>
         </widget>
        </item>
        <item row="0" column="5">
         <widget class="QPushButton" name="pushButton_21">
          <property name="text">
           <string>asinh</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QPushButton" name="pushButton_28">
          <property name="text">
           <string>ln</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QPushButton" name="pushButton_25">
          <property name="text">
           <string>log10</string>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QPushButton" name="pushButton_13">
          <property name="text">
           <string>cos</string>
          </property>
         </widget>
        </item>
        <item row="1" column="3">
         <widget class="QPushButton" name="pushButton_16">
          <property name="text">
           <string>acos</string>
          </property>
         </widget>
        </item>
        <item row="1" column="4">
         <widget class="QPushButton" name="pushButton_19">
          <property name="text">
           <string>cosh</string>
          </property>
         </widget>
        </item>
        <item row="1" column="5">
         <widget class="QPushButton" name="pushButton_22">
          <property name="text">
           <string>acosh</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QPushButton" name="pushButton_29">
          <property name="text">
           <string>exp</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QPushButton" name="pushButton_26">
          <property name="text">
           <string>log</string>
          </property>
         </widget>
        </item>
        <item row="2" column="2">
         <widget class="QPushButton" name="pushButton_14">
          <property name="text">
           <string>tan</string>
          </property>
         </widget>
        </item>
        <item row="2" column="3">
         <widget class="QPushButton" name="pushButton_17">
          <property name="text">
           <string>atan</string>
          </property>
         </widget>
        </item>
        <item row="2" column="4">
         <widget class="QPushButton" name="pushButton_20">
          <property name="text">
           <string>tanh</string>
          </property>
         </widget>
        </item>
        <item row="2" column="5">
         <widget class="QPushButton" name="pushButton_23">
          <property name="text">
           <string>atanh</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QPushButton" name="pushButton_11">
          <property name="text">
           <string>rint</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QPushButton" name="pushButton_30">
          <property name="text">
           <string>abs</string>
          </property>
         </widget>
        </item>
        <item row="3" column="2">
         <widget class="QPushButton" name="pushButton_31">
          <property name="text">
           <string>min</string>
          </property>
         </widget>
        </item>
        <item row="3" column="3">
         <widget class="QPushButton" name="pushButton_32">
          <property name="text">
           <string>max</string>
          </property>
         </widget>
        </item>
        <item row="3" column="4">
         <widget class="QPushButton" name="pushButton_33">
          <property name="text">
           <string>sum</string>
          </property>
         </widget>
        </item>
        <item row="3" column="5">
         <widget class="QPushButton" name="pushButton_34">
          <property name="text">
           <string>avg</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QGroupBox" name="groupBox_3">
       <property name="title">
        <string>Operators</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_6">
        <item row="0" column="0">
         <widget class="QPushButton" name="pushButton_45">
          <property name="text">
           <string>+</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QPushButton" name="pushButton_46">
          <property name="text">
           <string>-</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QPushButton" name="pushButton_47">
          <property name="text">
           <string>*</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QPushButton" name="pushButton_48">
          <property name="text">
           <string>/</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QPushButton" name="pushButton_49">
          <property name="text">
           <string>^</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QPushButton" name="pushButton_50">
          <property name="text">
           <string>==</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QPushButton" name="pushButton_51">
          <property name="text">
           <string>&gt;</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QPushButton" name="pushButton_52">
          <property name="text">
           <string>&lt;</string>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QPushButton" name="pushButton_53">
          <property name="text">
           <string>and</string>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QPushButton" name="pushButton_54">
          <property name="text">
           <string>xor</string>
          </property>
         </widget>
        </item>
        <item row="2" column="2">
         <widget class="QPushButton" name="pushButton_55">
          <property name="text">
           <string>not</string>
          </property>
         </widget>
        </item>
        <item row="3" column="2">
         <widget class="QPushButton" name="pushButton_56">
          <property name="text">
           <string>.</string>
          </property>
         </widget>
        </item>
        <item row="2" column="3" rowspan="2">
         <widget class="QPushButton" name="pushButton_2">
          <property name="text">
           <string>_e</string>
          </property>
         </widget>
        </item>
        <item row="0" column="3" rowspan="2">
         <widget class="QPushButton" name="pushButton_1">
          <property name="text">
           <string>_pi</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionExit"/>
   <addaction name="separator"/>
   <addaction name="actionAbout"/>
   <addaction name="actionHelp"/>
  </widget>
  <action name="actionExit">
   <property name="icon">
    <iconset resource="qmathparser.qrc">
     <normaloff>:/qmp/resource/exit.png</normaloff>:/qmp/resource/exit.png</iconset>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="icon">
    <iconset resource="qmathparser.qrc">
     <normaloff>:/qmp/resource/about.png</normaloff>:/qmp/resource/about.png</iconset>
   </property>
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionHelp">
   <property name="icon">
    <iconset resource="qmathparser.qrc">
     <normaloff>:/qmp/resource/help.png</normaloff>:/qmp/resource/help.png</iconset>
   </property>
   <property name="text">
    <string>Help</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="qmathparser.qrc"/>
 </resources>
 <connections/>
</ui>
